mqtt/mqtt-generic/src/settings.rs

```diff
@@ -204,13 +208,6 @@ mod tests {
         );
     }
 
-    #[test]
-    fn it_refuses_persistence_with_no_file_path() {
-        let settings = Settings::from_file(Path::new("test/config_no_file_path.json"));
-
-        assert_matches!(settings, Err(_));
-    }
-
```

Comment:
Now that we have defaults this test doesn't make sense

Reply:
Agree. I believe we don't need `test/config_no_file_path.json` file anymore